### PR TITLE
fixed broken xtext repo regarding xtext/xbase.ui.testing

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
@LorenzoBettini @dhuebner can one of you two guys please review this

File: releng/org.eclipse.xtend.sdk.feature/feature.xml [hunk 1]
```diff
@@ -174,5 +174,32 @@ http://www.eclipse.org/legal/epl-v10.html Description here.
...
+
+   <plugin
+         id="org.eclipse.xtext.xbase.ui.testing"
+         download-size="0"
+         install-size="0"
+         version="0.0.0"
+         unpack="false"/>
+
+   <plugin
+         id="org.eclipse.xtext.xbase.ui.testing"
```
Comment (dhuebner) [inline releng/org.eclipse.xtend.sdk.feature/feature.xml]: Why are the plugins added twice?

Comment (cdietrich) [inline releng/org.eclipse.xtend.sdk.feature/feature.xml]: cause i am a poor copy & paster

Comment (cdietrich) [inline releng/org.eclipse.xtend.sdk.feature/feature.xml]: @dhuebner i updated the pull request

Comment (dhuebner) [inline releng/org.eclipse.xtend.sdk.feature/feature.xml]: @cdietrich Feel free to merge. Xcore and 7 languages are waiting for new plugins

**Pull request merged**